### PR TITLE
drop Python 3.3, update to use 3.6

### PR DESCRIPTION
**Pull request by maartenbreddels**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -90,7 +90,6 @@
     # -- SUPPORT UNIFORM-WHEELS: Extra packages for Python 2.7, 3.3
```
Comment (blink1073) [inline setup.py]: this line mentions 3.3 as well

**Comment by blink1073** [comment]
This is another option: https://github.com/ipython/ipykernel/blob/master/setup.py#L105.

@minrk, I'd say dropping support for 3.3 is fine, but I'll defer to you here.

**Comment by blink1073** [comment]
The end of life passed two months ago: https://www.python.org/dev/peps/pep-0398/#x-end-of-life

**Comment by maartenbreddels** [comment]
I think this is a fluke, it had green lights before

**Comment by minrk** [comment]
yup, fine. Don't forget to set `requires_python` when dropping Python support.

**Comment by blink1073** [comment]
@maartenbreddels, for reference: https://packaging.python.org/tutorials/distributing-packages/#python-requires

**Comment by minrk** [comment]
Thanks!

**Comment by maartenbreddels** [comment]
👍

**Pull request merged**